scripts/swo_parser.py

```diff
@@ -121,18 +121,18 @@ def parse_itm_bytes(self, bstring):
                 bstring = bstring[1:]
                 continue
 
-            payload_size = 2**(header & 0x03 - 1)
+            payload_size = 2 ** (header & 0x03 - 1)
```

Comment:
ran my autoformatter on this file for readability. We really should get a python formatter in the loop....

Reply:
Which formatter did you use?

Reply:
just used the clion formatter. in the future we should standardize, but i just ranit to clean up some spacing